### PR TITLE
Add option for custom s3 endpoint

### PR DESCRIPTION
**Pull request by gueux**
Additionally for custom S3 endpoint into Near Lake Indexer https://github.com/near/near-lake/pull/29
It world be great to have the same feature for near-lake-framefork. That allow users to setup fully cloud agnostic environment through Localstack or Minio S3 compatible server.

### CODE REVIEW

**Comment by khorolets** [comment]
@gueux I guess the comments to the PR would be almost the same as on `near-lake` side, so you can fix the code after we've finished with `near-lake` PR https://github.com/near/near-lake/pull/29

**Comment by gueux** [comment]
@khorolets Hi, i've updated pull request in order to compatible with the original https://github.com/near/near-lake/pull/29 PR
Will be waiting for your feedback.

File: src/types.rs [hunk 1]
```diff
@@ -2,6 +2,8 @@ pub type BlockHeight = u64;
 
 /// Configuration struct for NEAR Lake Framework
 pub struct LakeConfig {
+    /// AWS S3 Custom Endpoint
```
Comment (khorolets) [inline src/types.rs:5]:
```suggestion
    /// AWS S3 compatible custom endpoint
```

**Review by khorolets — COMMENTED** [review]
Thanks! A few things and we are good to go

**Comment by khorolets** [comment]
@gueux and run `cargo fmt` before commit, please

**Comment by gueux** [comment]
added few changes as requested

**Review by khorolets — APPROVED** [review]

**Pull request merged**